src/compiler/types.ts

```diff
@@ -8547,6 +8547,7 @@ namespace ts {
         readonly providePrefixAndSuffixTextForRename?: boolean;
         readonly includePackageJsonAutoImports?: "auto" | "on" | "off";
         readonly provideRefactorNotApplicableReason?: boolean;
+        readonly jsxSnippetCompletion?: "auto" | "braces" | "none";

```

Comment:
I would suggest `jsxAttributeCompletionStyle` since (a) JSX is a big space and we might add other snippets in it, and (b) this name will be user-facing if you name it the same thing on the VS Code side, which is typical.

Reply:
Makes sense. Done.